src/redux/store.js

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { configureStore } from '@reduxjs/toolkit';
+import caseSlice from './cases/caseSlice';
+import deathSlice from './deaths/deathSlice';
+
+const store = configureStore({
+  reducer: {
+    case: caseSlice,
+    death: deathSlice,
+  },
+});
+
```

Comment:
you have disabled eslint and disabling linters is against Microverse policy because linters help us write clean code it would be nice if you enabled eslint in this file. kindly do this in all files were eslint has been disabled.